promscale/installation/docker.md

```diff
@@ -56,29 +57,32 @@ packages and instructions, see the
 
 Previously, our recommended image was located at [`timescaledev/promscale-extension`](https://hub.docker.com/r/timescaledev/promscale-extension).
 It was based on the [Alpine docker image for PostgreSQL](https://github.com/docker-library/postgres/blob/e8ebf74e50128123a8d0220b85e357ef2d73a7ec/12/alpine/Dockerfile).
-Because of [collation bugs](https://github.com/docker-library/postgres/issues/327) and other issues we have now switched our recommendation to the Debian-based image above.
-
-Our previous Alpine-based image will continue to be updated and supported until the end of 2022 but we encourage users to migrate to the `timescale/timescaledb-ha`. All new installations should switch to the `timescale/timescaledb-ha`image.
-
-You can also migrate to Debian version by doing the following (please note: this can be a lengthy process and involves downtime):
+Because of [collation bugs](https://github.com/docker-library/postgres/issues/327) and other issues we have now switched to the Debian-based image above.
```

Comment:
```suggestion
Because of [collation bugs](https://github.com/docker-library/postgres/issues/327) in the previous version, you should use the Debian-based image instead.
```

Reply:
I've edited this, but I'm not sure I like talking about things that are no longer true. Is it possible to not mention the old Alpine image, and just point to the new Debian one?

Reply:
I agree!!!! never a good practice to document issues.